PFM support broken

```
In file included from lo2s/src/perf/event_provider.cpp:27:
lo2s/include/lo2s/perf/pfm.hpp:25:10: fatal error: lo2s/perf/event_description.hpp: No such file or directory
   25 | #include <lo2s/perf/event_description.hpp>
      |       
```

Seems it was broken by c30368affbbaab019db60d844c8cae19b753f7bd

I suppose we'd like to have a PFM build in the CI.

--- a/include/lo2s/perf/event.hpp
+++ b/include/lo2s/perf/event.hpp
@@ -291,11 +291,14 @@ public:
 
     ~EventGuard()
     {
-        close(fd_);
+        if (fd_ != -1)
+        {
+            close(fd_);
+        }
     }
 
 protected:
-    int fd_;
+    int fd_ = -1;
 };
 
 } // namespace perf

--- a/include/lo2s/perf/pfm.hpp
+++ b/include/lo2s/perf/pfm.hpp
@@ -21,8 +21,7 @@
 
 #pragma once
 
-#include "event_provider.hpp"
-#include <lo2s/perf/event_description.hpp>
+#include <lo2s/perf/event.hpp>
 
 #include <cstring>
 #include <optional>
@@ -57,7 +56,7 @@ public:
         pfm_terminate();
     }
 
-    std::optional<EventDescription> pfm4_read_event(const std::string& ev_desc) const
+    std::optional<Event> pfm4_read_event(const std::string& ev_desc) const
     {
         pfm_perf_encode_arg_t arg;
         struct perf_event_attr attr;
@@ -73,10 +72,9 @@ public:
             return std::nullopt;
         }
 
-        EventDescription ev =
-            EventDescription(ev_desc, (perf_type_id)attr.type, attr.config, attr.config1);
+        Event ev = Event(ev_desc, (perf_type_id)attr.type, attr.config, attr.config1);
 
-        if (!event_is_openable(ev))
+        if (!ev.event_is_openable())
         {
             return std::nullopt;
         }
@@ -84,9 +82,9 @@ public:
         return ev;
     }
 
-    std::vector<EventDescription> get_pfm4_events() const
+    std::vector<Event> get_pfm4_events() const
     {
-        std::vector<EventDescription> events;
+        std::vector<Event> events;
 
         pfm_pmu_info_t pinfo;
         pfm_event_info_t info;

